xhprof_lib/config.sample.php

```diff
@@ -11,6 +11,7 @@
 $_xhprof['servername'] = 'myserver';
 $_xhprof['namespace'] = 'myapp';
 $_xhprof['url'] = 'http://url/to/xhprof/xhprof_html';
+$_xhprof['getparam'] = "_profile";
```

Comment:
1. This is BC break, because after `git pull` on a library people that already have their own config won't inherit this setting.
2. Also no point in using double quotes. Single quotes will do quite fine.